toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/GeospatialTrackingCameraController.kt

```diff
@@ -163,6 +163,16 @@ internal class GeospatialTrackingCameraController(
         }
     }
 
+    override fun getPointFromPose(pose: Pose, session: Session): Point? {
+        return try {
```

Comment:
Could make this an expression...
```suggestion
    override fun getPointFromPose(pose: Pose, session: Session): Point? = try {
```

Reply:
How about:
```kotlin
override fun getPointFromPose(pose: Pose, session: Session): Point? = try {
            session.earth?.getGeospatialPose(pose)?.let { geospatialPose ->
                Point(geospatialPose.longitude, geospatialPose.latitude, geospatialPose.altitude, WorldScaleParameters.SR_WGS84_WGS_VERTICAL).let {
                    GeometryEngine.projectOrNull(it, WorldScaleParameters.SR_CAMERA)
                }
            }
        } catch (e: NotTrackingException) {
            null
        }
```

Reply:
I made this change 👍